package.json

```diff
@@ -138,6 +144,8 @@
     "jasmine-core": "4.5.0",
```

Comment:
Do we still need to be specifying jasmine dependencies when we are using jest?

Reply:
I haven't replaced jasmine everywhere but just for one project (@fundamental-ngx/nx-plugin) so we still have to keep them both.